File: Domain/Domain/Job.cs
using System;

namespace Domain
{
    public class Job
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Entered { get; set; }
        public string Duration { get; set; }
        public string Status { get; set; }
        public string Requirements { get; set; }
        public string Notes { get; set; }
        public string Location { get; set; }
        public string Contacts { get; set; }
    }
}

File: Presentation/WebUI/Interviews/src/app/jobs/job.ts
export class Job {
  constructor(
   public Id: number,
   public Name: string,
   public Description:string,
   public DateEntered:any,
   public Duration:string,
   public JobStatusId:number,
   public Requirements:string,
   public Notes:string,
   public LocationId:number,
  ){}
}